doc/tutorial/sequence_file/sequence_file_solution3.cpp

```diff
@@ -67,21 +67,24 @@ int main()
 
     auto length_filter = std::views::filter([] (auto const & rec)
     {
-        return std::ranges::size(seqan3::get<seqan3::field::seq>(rec)) >= 5;
+        return std::ranges::size(rec.sequence()) >= 5;
     });
 
     // you can use a for loop
 
     // for (auto & rec : fin | length_filter | std::views::take(2))
     // {
-    //     seqan3::debug_stream << "ID: " << seqan3::get<seqan3::field::id>(rec) << '\n';
+    //     seqan3::debug_stream << "ID: " << rec.id() << '\n';
     // }
 
     // But you can also do this to retrieve all IDs into a vector:
     std::vector<std::string> ids = fin
                                  | length_filter                                    // apply length filter
                                  | std::views::take(2)                              // take first two records
-                                 | seqan3::views::get<seqan3::field::id>            // select only ID from record
+                                 | std::views::transform([](auto && record)         // select only ID from record
+                                   {
+                                       return record.id();
+                                   })
```

Comment:
I could also write

```suggestion
                                 | std::views::transform(&record::id)               // select only ID from record
                                 // this is the same as writing:
//                                 | std::views::transform([](auto && record)         
//                                   {
//                                       return record.id();
//                                   })
```

I wasn't sure if that is to advanced, we could add it here as a education

Reply:
Upps, I would need a type declaration: `using record_t = decltype(fin)::record_type`

or use `| std::views::transform(&decltype(fin)::record_type::id) ` directly.